app/views/shared/_completed_transaction_feedback.html.erb

```diff
@@ -0,0 +1,58 @@
+<% content_for :extra_headers do %>
+  <meta name="robots" content="noindex, nofollow" />
+<% end %>
+<% content_for :title do %>
+  <%= publication.title %>
+<% end %>
+  <% unless show_survey? %>
+    <p class="govuk-body"><%= t("controllers.contact.govuk.service_feedback.thanks_for_visiting") %></p>
+  <% end %>
+  <% if publication.promotion %>
+    <div class="promotion">
```

Comment:
I wonder if a [Notice](https://components.publishing.service.gov.uk/component-guide/notice) would work here 🤔

Reply:
ah I was just copying the previous design. I could ask about it and do as a separate PR?